src/services/DfusionService.ts

```diff
@@ -312,14 +313,20 @@ export class DfusionRepoImpl implements DfusionService {
         this._getTcr(),
       ])
 
+      const localOverride = TOKEN_OVERRIDES[networkId][tokenAddress.toLowerCase()]
+      if (localOverride) {
+        log.info(`Local override found for token address ${tokenAddress} on network ${networkId}:`, localOverride)
+      }
+
       const tokenJson = tokenList.find(token => token.addressByNetwork[networkId] === tokenAddress)
       token = {
-        symbol,
-        name,
         ...tokenJson,
-        decimals: decimals as number,
+        symbol: localOverride?.symbol || symbol,
+        name: localOverride?.name || name,
+        decimals: localOverride?.decimals || (decimals as number),
         address: tokenAddress,
         known: !!tokenJson || (tcr.size > 0 && tcr.has(tokenAddress)),
+        forceAddressDisplay: localOverride?.forceAddressDisplay,
       }
```

Comment:
Can do
```js
token = {
        symbol,
        name,
        ...tokenJson,
        ...localOverride,
        address: tokenAddress,
        known: !!tokenJson || (tcr.size > 0 && tcr.has(tokenAddress)),
      }
```
unless you expect `localOverride` to have something you don't want to spread all over `token`. Undefined, null, etc. spreads just fine, like :butter: